transition-scripts/values/values.yaml

```diff
@@ -1,10 +1,17 @@
 image:
   repository: ghcr.io/bcgov/sso
-  tag: 7.6.5-build.30
-  pullPolicy: IfNotPresent
+  tag: 24.0.6-build.1
+  pullPolicy: Always
```

Comment:
I wonder about keeping this at IfNotPresent, for the actual tagged releases we don't really want the underlying image to change ever, unlike a `dev` tag or something